doc/man/8/rbd.rst

```diff
@@ -478,6 +482,11 @@ To map an image via the kernel with cephx enabled::
 
        rbd map mypool/myimage --id admin --keyfile secretfile
 
+To map an image via the kernel with different cluster name other than default *ceph*.
+Here keyfile would be ceph1.client.admin.keyring and ceph configuration file would be ceph1.conf::
```

Comment:
I would remove this line since it muddles the example which is just showing connecting to cluster ```ceph1```